### DISCUSSION TITLE
OSS Weather TestFlight Build

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#183] [Ideas]
TestFlight Build for beta testing of app versions

**Comment by farfromrefug** [comment]
@jacobgonzales20 it is already available  . but my process is to automatically put the app in review as soon as I add a build to test flight. 
Also I already get bug reports from production builds. So the only thing that test flight would bring is to have less bug prone production builds. 
But for now I prefer it that way. Less time pushing new builds, and more Eco friendly( less build time, less upload)
